os/kernel/binary_manager/binary_manager_binfile.c

```diff
@@ -178,7 +179,7 @@ int binary_manager_create_entry(int requester_pid, char *bin_name, int version)
 		}
 	} else {
 		/* If it it not registered, register it */
-		ret = binary_manager_register_ubin(bin_name);
+		ret = binary_manager_register_ubin(bin_name, SCHED_PRIORITY_MIN);
```

Comment:
And we already checked that `bin_name` is already exist or not from `binary_manager_get_index_with_name`.
But in `binary_manager_register_ubin`, we checked again.
I think these are duplicated.